[FR] Add `Modding/Beatmap Nominator Tiers`, `BADASS`, `NAT meetings`, `QAT Leaders`, `Quality Assurance Team`, `Timestamp`, update `Modding` and `Kudosu` (#5724)
ppy/osu-wiki: wiki/Modding/fr.md

# Modding

*Pour d'autres utilisations, voir [Mod (désambiguïsation)](/wiki/Disambiguation/Mod).*

> Le **modding** est l'acte de fournir des commentaires sur la [beatmap](/wiki/Beatmap) d'un autre utilisateur. Un utilisateur qui fait du modding est appelé un *modder*, et ses messages collectifs sur la discussion d'un beatmap sont appelés un *mod*.

Reviewer: ```suggestion
Le **modding** est l'acte de fournir des commentaires sur la [beatmap](/wiki/Beatmap) d'un autre utilisateur. Un utilisateur qui fait du modding est appelé un *moddeur*, et ses messages collectifs sur la discussion d'une beatmap sont appelés un *mod*.
```

Les mods sont principalement utilisés pour améliorer les beatmaps visant la catégorie [classée](/wiki/Beatmap/Category#ranked). Parce qu'il s'agit d'un processus détaillé et analytique, le modding de beatmaps est également un moyen populaire pour les utilisateurs d'améliorer leur compréhension du beatmapping.

## Trouver des beatmaps à modder

Bien qu'il existe d'innombrables façons de découvrir les beatmaps à modder, voici quelques méthodes courantes :

- Le forum [Modding Queues](https://osu.ppy.sh/community/forums/60) est la plateforme la plus utilisée pour trouver des beatmaps à modder et demander des mods. Un utilisateur peut créer son propre fil de discussion, appelé "file d'attente", pour que d'autres utilisateurs puissent poster des maps à modder.

>   Le créateur de chaque file d'attente en dicte les règles, contrôlant quels beatmaps sont viables pour recevoir des mods. Les files d'attente sont généralement ouvertes aux demandes de deux types de mods :

Reviewer: ```suggestion
  Le créateur de chaque file d'attente en dicte les règles, contrôlant quelles beatmaps sont viables pour recevoir des mods. Les files d'attente sont généralement ouvertes aux demandes de deux types de mods :
```

  - **Mod normal (NM):** Un mod qui sera fourni sans travail de la part du mappeur demandeur.
  - **Mod for mod (M4M):** Un échange qui nécessite que chaque utilisateur mod la beatmap de l'autre.

- Les moddeurs connus indiquent souvent s'ils acceptent ou non les demandes de mod sur leur profil d'utilisateur.

> - Il n'est **pas** recommandé d'effectuer des recherches dans la liste des beatmaps en attente pour trouver des beatmaps à modder, car la plupart des soumissions ne visent pas le statut classée, et leurs créateurs ne répondront donc potentiellement pas aux mods.

Reviewer: ```suggestion
- Il n'est **pas** recommandé d'effectuer des recherches dans la liste des beatmaps en attente pour trouver des beatmaps à modder, car la plupart des beatmaps publiées ne visent pas le statut classé, et leurs créateurs ne répondront donc potentiellement pas aux mods.
```

## Comment modder

> Pour modder un beatmap, commencez par télécharger la version la plus récente et ouvrez sa [page de discussion](/wiki/Beatmap_Discussion). Jouez à la [difficulté](/wiki/Beatmap/Difficulty) que vous souhaitez modifier et gardez à l'esprit tout problème lié au gameplay. Lorsque vous avez terminé, ouvrez la difficulté dans l'éditeur et analysez-la à la recherche d'autres problèmes.

Reviewer: ```suggestion
Pour modder une beatmap, commencez par télécharger la version la plus récente et ouvrez sa [page de discussion](/wiki/Beatmap_Discussion). Jouez à la [difficulté](/wiki/Beatmap/Difficulty) que vous souhaitez modifier et gardez à l'esprit tout problème lié au gameplay. Lorsque vous avez terminé, ouvrez la difficulté dans l'éditeur et analysez-la à la recherche d'autres problèmes.
```

Lorsque vous rencontrez un problème, sélectionnez tous les objets pertinents, puis utilisez le raccourci de copie (`Ctrl` + `C`) pour copier leurs informations d'horodatage en tant que texte. Sur la discussion d'une beatmap, collez (`Ctrl` + `V`) les horodatages applicables et vos commentaires. L'onglet `Chronologie` est utilisé pour les problèmes concernant des parties spécifiques d'une beatmap, l'onglet `Général (cette difficulté)` est utilisé pour les commentaires sur de plus grandes sections d'une beatmap, et l'onglet `Général (Toutes les difficultés)` est utilisé pour les problèmes applicables à toutes les difficultés d'une beatmap.

> Tous les messages de mods doivent traiter de la nature du problème, de sa raison d'être et de la manière de le résoudre. Voir le [Code de conduite pour le modding et le mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping#making-a-mod-post) pour plus de détails sur la façon de formater les messages de mod.

Reviewer 2: Je trouvais que "formater" faisait un peu trop informatique et je propose ça comme alternative au cas où
```suggestion
Tous les messages de mods doivent traiter de la nature du problème, de sa raison d'être et de la manière de le résoudre. Voir le [Code de conduite pour le modding et le mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping#making-a-mod-post) pour plus de détails sur la mise en forme des messages de mod.
```

> Les mappeurs fournissent régulièrement des [kudosu](/wiki/Modding/Kudosu) aux messages des modder qu'ils considèrent comme utiles en cliquant sur le bouton de vote "like". Cette statistique est enregistrée sur le profil d'un individu et est référencée lors de la demande d'adhésion aux [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators).

Reviewer: ```suggestion
Les mappeurs fournissent régulièrement des [kudosu](/wiki/Modding/Kudosu) aux messages des moddeurs qu'ils considèrent comme utiles en cliquant sur le bouton de vote "like". Cette statistique est enregistrée sur le profil de l'utilisateur et est référencée lors de la demande d'adhésion aux [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators).
```

> ## Types de postes des modders

Reviewer: ```suggestion
## Types de postes des moddeurs
```

L'objectif de chaque message de type `Problème` ou `Suggestion` sur une page de discussion d'une beatmap est d'expliquer tout ce qui concerne une beatmap dans le contexte de sa conception.

De nombreux messages de mods traitent des erreurs dans une beatmap. Par exemple, l'absence d'un son de clap qui serait attendu par le schéma de hitsounding de la beatmap est probablement une erreur.

> Proposer des méthodes alternatives de beatmapping est un autre type de post de mod courant. Si une partie d'une beatmap peut être acceptable dans sa forme actuelle, un moddeur peut expliquer comment l'améliorer encore en le concevant différemment.

Reviewer 2: ```suggestion
Proposer des méthodes alternatives de beatmapping est un autre type de post de mod courant. Si une partie d'une beatmap peut être acceptable dans sa forme actuelle, un moddeur peut expliquer comment l'améliorer encore en la concevant différemment.
```

Reviewer 2: Je propose ça comme alternative parce que la fin de la phrase me semble un peu bizarre.
```suggestion
Proposer des méthodes alternatives de beatmapping est un autre type de post de mod courant. Si une partie d'une beatmap peut être acceptable dans sa forme actuelle, un moddeur peut expliquer comment l'améliorer encore en la modifiant.
```

> Les beatmaps qui visent la catégorie classée doivent impérativement respecter les [critères de classement](/wiki/Ranking_Criteria). Un certain nombre de messages de mods traitent des problèmes cités directement dans ces critères.

Reviewer 3: ```suggestion
Les beatmaps qui visant la catégorie classée doivent impérativement respecter les [critères de classement](/wiki/Ranking_Criteria). Un certain nombre de messages de mods traitent des problèmes cités directement dans ces critères.
```
C'est un changement mineur mais personnellement je trouve que ça rend mieux

Author: Bah la je trouve qu'il y a une faute d'orthographe, c'est bien comme c'est à mon avis

Reviewer 3: j'ai oublié d'enlever le "qui" 😓

> Les beatmaps qui visent la catégorie classée doivent impérativement respecter les [critères de classement](/wiki/Ranking_Criteria). Un certain nombre de messages de mods traitent des problèmes cités directement dans ces critères.

Author: ```suggestion
Les beatmaps visant la catégorie classée doivent impérativement respecter les [critères de classement](/wiki/Ranking_Criteria). Un certain nombre de messages de mods traitent des problèmes cités directement dans ces critères.
```
